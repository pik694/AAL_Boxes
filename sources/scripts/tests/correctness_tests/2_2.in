2
46 2 68
1 99 13

