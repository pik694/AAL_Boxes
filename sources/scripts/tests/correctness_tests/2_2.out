1 0 
7543->7543
